inc/structured-data-blocks/class-how-to-block.php

```diff
@@ -78,38 +78,59 @@ protected function get_json_ld( array $attributes ) {
 			$json_ld['step'] = array();
 			$steps = array_filter( $attributes['steps'], 'is_array' );
 			foreach ( $steps as $index => $step ) {
-				$json_ld['step'][] = $this->get_step_json_ld( $step, $index );
+				$json_ld['step'][] = $this->get_section_json_ld( $step, $index );
 			}
 		}
 
 		return $json_ld;
 	}
 
 	/**
-	 * Returns the JSON LD for a step in a how-to block in array form.
+	 * Returns the JSON LD for a step text (description) in a how-to block in array form.
 	 *
-	 * @param array $step  The attributes of a step in the how-to block.
-	 * @param int   $index The index of the step in the how-to block.
+	 * @param array $step  The attributes of a step(-section) in the how-to block.
+	 * @param int   $index The index of the section in the how-to block.
 	 *
-	 * @return array The JSON LD representation of the step in a how-to block in array form.
+	 * @return array The JSON LD representation of the step text (description) in a how-to block in array form.
```

Comment:
I'd say that you can remove the `[...] in a how-to block in array form` part from this line as it's redundant.